tests/conftest.py

```diff
@@ -35,7 +35,9 @@ def app():
 
 @pytest.fixture
 def testapp(app):
-    return TestApp(app)
+    _app = TestApp(app)
+    _app.authorization = ('Basic', ('username', 'password'))
```

Comment:
Put another way, someone coming to look at this app in the future (like one of us) may look at one of these methods being tested and not immediately realize it requires authentication. If the test explicitly adds `testapp.authorization = ('Basic', ('username', 'password'))` there is less chance of not understanding what methods need auth by quickly looking at tests.